consumer/src/main/kotlin/streams/kafka/KafkaEventSink.kt

```diff
@@ -108,6 +115,7 @@ class KafkaEventSink(private val config: Map<String, String>,
         StreamsUtils.ignoreExceptions({
             runBlocking {
                 if (job.isActive) {
+                    eventConsumer.wakeup()
```

Comment:
I know we're not concerned with WakeupException here based on what it means.  But if it happens -- then job.cancelAndJoin() doesn't happen.  Is that OK?  Watch out for that possibility.

Reply:
The Kafka documentation states:
```
/**
  * Wakeup the consumer. This method is thread-safe and is useful in particular to abort a long poll.
  * The thread which is blocking in an operation will throw {@link org.apache.kafka.common.errors.WakeupException}.
  * If no thread is blocking in a method which can throw {@link org.apache.kafka.common.errors.WakeupException}, the next call to such a method will raise it instead.
  */
```

So the `WakeupException` should be thrown into the coroutine, here:

https://github.com/neo4j-contrib/neo4j-streams/blob/56e99d5bd38851020c457b15a99cec7319cd3e48/consumer/src/main/kotlin/streams/kafka/KafkaEventSink.kt#L149-L155